introduction.md

```diff
@@ -1 +1,8 @@
-What's good, codeschool homies? I'm into the iOS code path. That one looks 🏆.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?